sdk/js/index.js

```diff
@@ -99,8 +76,36 @@ const on = (message, f) => {
   return squad.connection.on(message, f)
 }
 
-const call = (method, data) => {
-  return squad.connection.call(method, data)
+const call = async (zome, method, inputs) => {
+  const info = await squad.connection.call('info/instances', {})
+  const instanceId = info[0].id
+
+  const params = {
+    "instance_id": instanceId,
+    "zome": zome,
+    "function": method,
+    "args": inputs
+  }
+
+  const result = JSON.parse(await squad.connection.call('call', params))
+
+  if (result.Ok === undefined) {
+    console.log(result)
+    throw result
+  }
+  return result.Ok
+}
+
+const createElement = async (element) => {
+  return await call("elements", "create_element", {"element": element})
+}
+
+const getElement = async (address) => {
+  return await call("elements", "get_element", {"address": address})
+}
+
+const getAllGames = async () => {
+  return await call("elements", "get_games", {})
```

Comment:
My instinct here is to name things int he SDK the same as the zome function and vice versa. 

So we would either name this `const getGames` (which feels like we might be able to specify a query)
or we would name the zome function `get_all_games` which makes more sense to me.

If we are committed to this we will be able to infer the sdk code when looking at the zome documentation and not have to have 2 docs open to cross reference.